VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -613,7 +618,21 @@ public override async Task RetrieveResources(SessionHostsStartInfo sessionHostsS
 
         public override async Task<IEnumerable<string>> List()
         {
-            return (await _dockerClient.Containers.ListContainersAsync(new ContainersListParameters())).Select(x => x.ID);
+            return (await _dockerClient.Containers.ListContainersAsync(
```

Comment:
We are sure this works fully well right? Because we are not putting this behind any flag.

Reply:
I meant in the cases we are tackling today, where a network part or something gets created. I think this is clever, thanks for thinking of it. I would just make sure that this works for the case we have seen. If there are any logs that you want to add to VMAgent, go for it. 
For example, we can start listing all containers, logs ones that are created by us and others that aren't and then , in another PR, kill only the ones that we created.

Reply:
chatted, we're good to go! thanks!